app/code/Magento/LoginAsCustomer/Test/Mftf/Test/AdminLoginAsCustomerLoggingFilterTest.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminLoginAsCustomerLoggingFilterTest">
+        <annotations>
+            <features value="Login as Customer logs"/>
+            <stories value="Filter by date login as customer logs"/>
+            <title value="Filter by date login as customer logs"/>
+            <description value="Filter by date should be from/to"/>
+            <severity value="MAJOR"/>
+            <group value="login_as_customer"/>
+            <testCaseId value="MC-"/>
+        </annotations>
+        <before>
+            <magentoCLI command="config:set {{LoginAsCustomerConfigDataEnabled.path}} 1"
+                        stepKey="enableLoginAsCustomer"/>
+            <magentoCLI command="config:set {{LoginAsCustomerStoreViewLogin.path}} 0"
+                        stepKey="enableLoginAsCustomerAutoDetection"/>
+            <magentoCLI command="cache:flush config" stepKey="flushCacheBeforeTestRun"/>
```

Comment:
I would strongly recommend invalidating cache instead of flushing. There's an action group that might help

https://github.com/magento/magento2/blob/e9190a5bb2c4aede2a752e94a3e87dcf9647b604/app/code/Magento/Backend/Test/Mftf/ActionGroup/CliCacheCleanActionGroup.xml#L11

Thanks

Reply:
The same recommendation applies to all places in the tests where the cache flush is being happened

Reply:
Have been updated